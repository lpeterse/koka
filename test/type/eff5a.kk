// test if we keep effects polymorph
// ok     : forall<e,a> (f: (string) -> e a) -> ((string) -> e a) with (partial<=e)
fun ok(f) = choose(error,f)
