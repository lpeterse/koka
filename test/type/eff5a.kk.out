ok      : forall<a,e> (f : (string) -> <exn|e> a) -> ndet ((string) -> <exn|e> a)
